unity-renderer/Assets/DCLPlugins/ECS7/ECSComponents/Utils/ECSComponentsUtils.cs

```diff
@@ -2,12 +2,35 @@
 using System.Collections.Generic;
 using DCL;
 using DCL.Configuration;
+using DCL.ECSComponents;
 using DCL.Helpers;
 using DCL.Models;
 using UnityEngine;
 
 public static class ECSComponentsUtils
 {
+    internal const string AVATAR_TRIGGER_LAYER = "AvatarTriggerDetection";
+    
+    public static HashSet<GameObject> DetectAvatars(in UnityEngine.Vector3 box, in UnityEngine.Vector3 center, in Quaternion rotation, in HashSet<Collider> excludeColliders = null)
+    {
+        Collider[] colliders = Physics.OverlapBox(center, box * 0.5f, rotation,
+            LayerMask.GetMask(AVATAR_TRIGGER_LAYER), QueryTriggerInteraction.Collide);
+
+        if (colliders.Length == 0)
+            return null;
+
+        bool hasExcludeList = excludeColliders != null;
+        HashSet<GameObject> result = new HashSet<GameObject>();
+        foreach (Collider collider in colliders)
+        {
+            if (hasExcludeList && excludeColliders.Contains(collider))
+                continue;
+            
+            result.Add(collider.transform.parent.gameObject);
+        }
+        return result;
+    }
```

Comment:
could we create an specific utils class for this? `ECSAvatarUtils` or something like that?
I would like if we can have topic-related utils instead of throwing unrelated functions in a single class